cmd/update/update.go

```diff
@@ -0,0 +1,24 @@
+package update
+
+import (
+	cmdcore "github.com/lyft/flytectl/cmd/core"
+
+	"github.com/spf13/cobra"
+)
+
+// CreateUpdateCommand will return update command
+func CreateUpdateCommand() *cobra.Command {
+	updateCmd := &cobra.Command{
+		Use:   "update",
+		Short: "Update various resources.",
+	}
+
+	updateResourcesFuncs := map[string]cmdcore.CommandEntry{
+		"project":    {CmdFunc: updateProjectsFunc, Aliases: []string{"projects"}, ProjectDomainNotRequired: true},
+	}
+
+	cmdcore.AddCommands(updateCmd, updateResourcesFuncs)
+	updateCmd.PersistentFlags().BoolVarP(&(GetConfig().ActivateProject), "activate", "t", false, "Activates the project specified as argument.")
```

Comment:
These flags should be only for the `project` command. Currently you have added them to all update commands, which is not correct

Reply:
this has to be changed

Reply:
Was waiting on my other MR to be merged. Took care of this.